build/docker/edgeimg-make.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+REPOSITORY=${REPOSITORY:-"kubeedge/edgecore"}
+RELEASE_VERSION=${RELEASE_VERSION:-"v1.9.1"}
```

Comment:
seems that we need update this version after each release?

Reply:
Should got from git tag :)